bin/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Run script used by fluo-dev & fluo-deploy
```

Comment:
It probably doesn't.  In fluo-io/fluo-deploy#62, I want to rename the test command to run.  I added a note to make sure argument passing works too.